### PR TITLE
Bees.

### PR DESCRIPTION
**Pull request by usufruct99**
I would like to add more bees.

@jrouly @rcmurphy

### CODE REVIEW

**Comment by rcmurphy** [comment]
Getting a content length exceeded on the GIF unfortunately.

File: LICENSE [hunk 1]
```diff
@@ -1,5 +1,5 @@
 MIT License
-Copyright (c) <year> <copyright holders>
```
Comment (jrouly) [inline LICENSE]: 🤦‍♂️

File: README.md [hunk 2]
```diff
@@ -15,3 +15,7 @@ Bees. Direct to you.
 ### Publishing locally
 
     $ sbt bees-direct-play/docker:publishLocal
+
+### Media mentions
+
+![Bees](https://i.imgur.com/SIKxCJ3.gif)
```
Comment (jrouly) [inline README.md]: 🐝

**Comment by jrouly** [comment]
> Getting a content length exceeded on the GIF unfortunately.

Oh I see what you mean @rcmurphy. Oops.

**Comment by usufruct99** [comment]
bummer.  y u hate 🐝 , github?

**Pull request merged**